db/migrate/20241008145926_e_bkiller/commits.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+
+# If you are creating only a single commit, you may instead use a single :all
+# field to have all of the CSV columns be included. This cannot be used when
+# creating multiple commits.
+- fields:
+  - :all
+  message:
+    Updating Data extraction method
```

Comment:
Can we make this more informative? Something to indicate that we exclude energy production keys from the EB method for EU datasets and set them through migration instead - while keeping the data values itself the same.